.editorconfig

```diff
@@ -0,0 +1,18 @@
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+	
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+
+[{.jshintrc,*.json,*.yml}]
```

Comment:
Shouldn't `*.json` by default be the same as the standards for JS, which is 1 tab? For JSON managed by a tool, like `package.json`, the code style should be the same as whatever the tool outputs, like `npm` will use 2 spaces for indentation.